chia/cmds/configure.py

```diff
@@ -237,6 +255,12 @@ def configure(
 )
 @click.option("--set-node-introducer", help="Set the introducer for node - IP:Port", type=str)
 @click.option("--set-farmer-peer", help="Set the farmer peer for harvester - IP:Port", type=str)
+@click.option("--set-solver-peer", help="Set the solver peer for farmer - IP:Port", type=str)
```

Comment:
If you have a setup with a farmer running on a separate machine, I imagine this has to be run on the farmer machine itself, right?